src/config.rs

```diff
@@ -2,12 +2,14 @@ use std::env;
 
 pub struct Config {
     pub database_url: String,
+    pub server_port: Number,
```

Comment:
This type doesnt exists, you probably want a `u16` here.

Also make sure you run `cargo check` and `cargo fmt`